index.js

```diff
@@ -121,6 +129,10 @@ module.exports = class Filter extends Plugin {
       loggerName += ' > [' + annotation + ']';
     }
 
+    FSMERGER.set(this, {
```

Comment:
Currently I am worried, that this plugin does not use `this.input`, did we make a mistake with it? By not using `this.input` we are making it hard for future optimizations, as ensuring all plugins go through `this.input` `this.output` opens many doors when it comes to future optimizations.

Reply:
We may have to make changes to the `fs-merger` library to somehow match inputPaths and input to `broccoli-persistent-filter` where we expect input can be `broccoli-node` `string` or an `object with prefix/getDestinationPath` function.